pages/common/vlc.md

```diff
@@ -18,3 +18,7 @@
 - Play repeatedly:
 
 `vlc --loop {{path/to/file}}`
+
+- Play video from the web:
+
+`vlc https://www.youtube.com/watch?v=oHg5SJYRHA0`
```

Comment:
```suggestion
- Play video from a URL:

`vlc {{https://www.youtube.com/watch?v=oHg5SJYRHA0}}`
```
I think this is more clear. Braces around the URL are there to show that the URL can be changed (i.e. you can put another YouTube link in the same place).